expression-language/4.0/_index.md

```diff
@@ -11,7 +11,8 @@ Jakarta Expression Language defines an expression language for Java applications
 * [Jakarta Expression Language 4.0 Specification Document](./jakarta-expression-language-spec-4.0.html) (HTML)
 * [Jakarta Expression Language 4.0 Javadoc](./apidocs)
 * [Jakarta Expression Language 4.0 TCK](https://download.eclipse.org/jakartaee/expression-language/4.0/jakarta-expression-language-tck-4.0.0.zip)  ([sig](https://download.eclipse.org/jakartaee/expression-language/4.0/jakarta-expression-language-tck-4.0.0.zip.sig),  [sha](https://download.eclipse.org/jakartaee/expression-language/4.0/jakarta-expression-language-tck-4.0.0.zip.sha256),  [pub](https://jakarta.ee/specifications/jakartaee-spec-committee.pub))
-   * Adds JDK 11 support [Jakarta Expression Language 4.0.1 TCK](https://download.eclipse.org/jakartaee/expression-language/4.0/jakarta-expression-language-tck-4.0.1.zip)  ([sig](https://download.eclipse.org/jakartaee/expression-language/4.0/jakarta-expression-language-tck-4.0.1.zip.sig),  [sha](https://download.eclipse.org/jakartaee/expression-language/4.0/jakarta-expression-language-tck-4.0.1.zip.sha256),  [pub](https://jakarta.ee/specifications/jakartaee-spec-committee.pub))
+  * For all TCK releases, see download directory here: https://download.eclipse.org/jakartaee/expression-language/4.0
+    * Adds JDK 11 support [Jakarta Expression Language 4.0.1 TCK](https://download.eclipse.org/jakartaee/expression-language/4.0/jakarta-expression-language-tck-4.0.1.zip)  ([sig](https://download.eclipse.org/jakartaee/expression-language/4.0/jakarta-expression-language-tck-4.0.1.zip.sig),  [sha](https://download.eclipse.org/jakartaee/expression-language/4.0/jakarta-expression-language-tck-4.0.1.zip.sha256),  [pub](https://jakarta.ee/specifications/jakartaee-spec-committee.pub))
```

Comment:
I thought that with this change, this bullet would go away completely and we would just have the original TCK version and then the new bullet you added and not require the service release ones to be listed.

From the meeting minutes from Jan 22:

```
Should there be an announcement for when service releases are published? The page itself would no longer be getting updated
- All versions are valid
- No
```
This implied that we wouldn't list the service releases any longer on the component pages which is what the original point was to make things be more uniform for the different component specifications.